lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -107,7 +110,7 @@ def emit(event)
         router.emit(@tag, record[0], record[1])
       else
         time = (event.timestamp / 1000).floor
-        record = JSON.parse(event.message)
+        record = Yajl.load(event.message)
```

Comment:
Standard JSON module has `#load` method.
We can add Decoder option like as:
https://github.com/choplin/fluent-plugin-pgjson/pull/13/commits/3e98da6d5fe201dd8c6877eb4c384bf943d7b41b

Reply:
> Standard JSON module has #load method.
> We can add Decoder option like as:
> choplin/fluent-plugin-pgjson@3e98da6

I'm kinda wondering: why do we need to make it plugglable which JSON library to use?

Provided that the mothership server (a.k.a. Fluentd core) is using YAJL to handle JSON
records, it seems to me that there is little interest to provide an option to users here.
Other parts of the system will use YAJL anyway, right?

Reply:
> ther parts of the system will use YAJL anyway, right?

Partially, yup. 
Yajl *and Oj* are used in almost Fluentd world for JSON encoder/decoder.
Fluentd prefers to use `Oj` if available for performance.

As of respecting [@ryotarai 's comment](https://github.com/fluent-plugins-nursery/fluent-plugin-cloudwatch-logs/pull/58#issuecomment-313266202), we might have to provide pluggable json decoder option.

Reply:
All right.

Now since this PR is conflicting with the master, I'll implement the parameter
you mentioned and submit the patch as another PR tomorrow.